apps/server/src/modules/learnroom/service/coursegroup.service.spec.ts

```diff
@@ -38,6 +38,42 @@ describe('CourseGroupService', () => {
 		jest.clearAllMocks();
 	});
 
+	describe('findAllCourseGroupsByUserId', () => {
+		describe('when finding by userId', () => {
+			const setup = () => {
+				const user = userFactory.buildWithId();
+				const courseGroup1 = courseGroupFactory.buildWithId({ students: [user] });
```

Comment:
Only a hint you can use 
const courseGroups = courseGroupFactory.buildListWithId(2, { students: [user] });

That give you the same result in one line, instead of 3.